libgrust/libproc_macro/Makefile.in

```diff
@@ -469,6 +470,7 @@ mostlyclean-compile:
 distclean-compile:
 	-rm -f *.tab.c
 
+@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/ident.Plo@am__quote@
```

Comment:
Those changes are generated and I'm indeed pushing them. The history might look a bit strange around those since there was multiple conflicts during some rebase and I had to manually fix those.